spec/models/student_loans/permitted_parameters_spec.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe StudentLoans::PermittedParameters do
+  let(:claim) { create(:tslr_claim) }
```

Comment:
Does this actually need to be a persisted claim? If not we should avoid creating and just pass in a `TslrClaim.new` to keep the specs snappy.